docs/roadmap/Roadmap.md

```diff
@@ -1,9 +1,67 @@
+
+## Web platform
+In 2024 we made strong progress on performance and quality (such as app size reduction, and better use of multi-threading and improved app load times).
+
+In 2025 we plan further improvements in the core of Flutter web, for example: accessibility, text input, international text rendering, size, performance, and platform integration. We also continue to work on improving Web performance using compilation to Wasm/WebAssembly.
+
+The new JS interop mechanism for Dart that supports both JS and Wasm compilation is considered complete. With that, in 2025 we plan on removing the legacy HTML and JS libraries (see breaking change announcement).
+
+Finally, we've made good progress on support for hot reload on the web, and hope to launch it in 2025.
+
+## Desktop platform
+The Flutter core team is focusing on mobile and web support in 2025, however we'll continue the collaboration initiated in 2024 with the Canonical team to drive support for multi-window on desktop platforms (Linux, macOS, and Windows).
+
+## Tooling and AI
+We'll continue integrate with AI solutions to offer AI assistance for core programming tasks.
+
+We'll continue to invest in our suite of tooling including Flutter DevTools, VS Code, Android Studio/IntelliJ, as well as IDX, with an eye toward always improving the edit-refresh cycle and the overall developer experience.
+
+## Dart programming language
+In 2024 we concluded that supporting macros in Dart was not viable. Based on that, in 2025 we expect to improve the current support for code generation in build_runner, and to investigate alternatives ways of improving Dart support for serialization and deserialization.
+
+We also expect to ship one or more language features currently going through the Dart language design funnel.
+
+## Dart compilers and tools
+We'll work on refactoring the Dart analyzer and the front-end compiler to share more of their implementations, with the goal of accellerating future language feature development, and to make them more performant and stable.
+
+We also plan on investigating support for cross-compiling Dart AOT executables (for example, compiling to a Linux AOT executable on a macOS development machine).
+
+## Releases
+We plan to have four stable releases and 12 beta releases during 2025, similar to 2024. To improve the predictability and regularity of these releases, and to reduce regressions in stable releases, we'll invest in further test coverage.
+
+We'll also make investments to improve our ability to expedite the release of any needed hotfix/patch  releases.
+
+## Non-goals
+We're still not planning on investing in built-in support for code push or hot updates. For code push, our friends at shorebird.dev may have offerings of interest. For UI push (also known as server-driven UI), we recommend the rfw package.
+
+We're also not planning on adding any additional supported platforms.
+
+
 # 2024
```

Comment:
Should we move the 2024 roadmap to the archive? https://github.com/flutter/flutter/blob/master/docs/roadmap/%5BArchive%5D-Old-Roadmaps.md